src/components/document-metadata-modal.js

```diff
@@ -272,32 +218,21 @@ function DocumentMetadataModal({
             placeholder={t('tagsPlaceholder')}
             />
         </FormItem>
-        {initialDocumentMetadata?.roomId && (
-          <FormItem label={t('dueOn')} name="dueOn">
-            <DatePicker
-              inputReadOnly
-              style={{ width: '100%' }}
-              format={dateTimeFormat}
-              disabledTime={disabledTime}
-              showTime={{ defaultValue: moment(`${firstEnabledHour}:00`, 'HH:mm'), format: 'HH:mm', hideDisabledOptions: true }}
-              />
-          </FormItem>
-        )}
-        <FormItem name="enableSequence" valuePropName="checked" hidden>
-          <Checkbox />
-        </FormItem>
         {mode === DOCUMENT_METADATA_MODAL_MODE.create && allowMultiple && (
-          <Collapse className="DocumentMetadataModal-sequenceCollapse" activeKey={isSequenceExpanded ? ['sequence'] : []} onChange={handleSequenceCollapseChange} ghost>
-            <CollapsePanel header={t('createSequence')} key="sequence" forceRender>
-              <Alert className="DocumentMetadataModal-sequenceInfo" message={t('sequenceInfoBoxHeader')} description={t('sequenceInfoBoxDescription')} />
-              <FormItem label={t('sequenceInterval')} name="sequenceInterval">
-                <Select options={documentSequenceIntervalOptions} disabled={!hasDueDate} />
-              </FormItem>
-              <FormItem label={t('sequenceCount')} name="sequenceCount" rules={[{ type: 'integer', min: 1, max: 100 }]}>
-                <InputNumber style={{ width: '100%' }} disabled={!hasDueDate} min={1} max={100} />
-              </FormItem>
-            </CollapsePanel>
-          </Collapse>
+          <FormItem
+            name="sequenceCount"
+            rules={[{ type: 'integer', min: 1, max: 100 }]}
+            label={
+              <Fragment>
+                <span>{t('createSequence')}</span>
```

Comment:
This was once the header ("Reihe anlegen"), now it is a label for a number input. I'm not sure if it will be clear to the users that they have to enter the number of documents there. Maybe we should at least mention in the hover info text that they can type there how many documents should be generated? :thinking:

Reply:
On second thought, maybe it is clear. I'm not sure here and maybe over-concerned. You can also just ignore this comment.

Reply:
I had a hard time deciding on this one too. Was not sure how to combine "Reihe anlegen" with "Anzahl". I think the info text does make it clear though. 
In lack of better ideas, let's just see what the others will think. I can actually leave a comment on the ticket for Theresa and ask her if she thinks it's clear enough for a user or has suggestions.

Reply:
:+1: